3696/3/app/helpers/auth_helper.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra/base'
+
+module Sinatra
```

Comment:
That is a really bad practice. You ARE polluting `Sinatra` namespace. Why? Just use your own!